packages/flutter_hooks/lib/src/debounced.dart

```diff
@@ -0,0 +1,74 @@
+part of 'hooks.dart';
+
+/// Returns a [ValueNotifier] that updates its value with a timeout
+/// for the given input [toDebounce] and triggers widget rebuilds accordingly.
+ValueNotifier<T?> useDebounced<T>(
+  T toDebounce, {
+  Duration timeout = const Duration(milliseconds: 500),
```

Comment:
I would make the duration required and positional. For a useDebounced, that makes sense I think.

 I want to avoid arbitrary durations in the framework.